panoptes_client/workflow.py

```diff
@@ -485,36 +485,37 @@ def add_alice_rules_and_effects(self, question_retirement_limit=3, count_retirem
         Count Subject Rule created will trigger retirement when the classification count reaches this limit
 
         """
-        question_subject_rule = self.add_rule(
+        question_subject_rule = self.add_caesar_rule(
             f'["gte", ["lookup", "complete.0", 0], ["const", {question_retirement_limit}]]',
             'subject'
         )
         send_to_alice_effect_config = {
             'url': 'https://tove.zooniverse.org/import',
             'reducer_key': 'alice'
         }
-        self.add_rule_effect('subject', question_subject_rule['id'], 'external', send_to_alice_effect_config)
-        self.add_rule_effect('subject', question_subject_rule['id'], 'retire_subject', {'reason': 'consensus'})
+        self.add_caesar_rule_effect('subject', question_subject_rule['id'], 'external', send_to_alice_effect_config)
+        self.add_caesar_rule_effect('subject', question_subject_rule['id'], 'retire_subject', {'reason': 'consensus'})
 
-        count_subject_rule = self.add_rule(
+        count_subject_rule = self.add_caesar_rule(
             f'["gte", ["lookup", "count.classifications", 0], ["const", {count_retirement_limit}]]',
             'subject'
         )
-        self.add_rule_effect('subject', count_subject_rule['id'], 'external', send_to_alice_effect_config)
-        self.add_rule_effect('subject', count_subject_rule['id'], 'retire_subject', {'reason': 'classification_count'})
+        self.add_caesar_rule_effect('subject', count_subject_rule['id'], 'external', send_to_alice_effect_config)
+        self.add_caesar_rule_effect('subject', count_subject_rule['id'], 'retire_subject', {'reason': 'classification_count'})
 
     def configure_for_alice(self):
         """
-        Configures workflow for ALICE/TOVE. This assumes workflow has alreaady been added to Caesar.
-        (Can do this via `workflow.add_to_caesar(True, True)`)
+        Configures workflow for ALICE/TOVE.
 
+        - This method will add workflow to Caesar
         - This method will create Caesar Extractors needed for ALICE with defaults.
         - This method will also create Caesar Reducers needed for ALICE with defaults.
         (In particular, `minimum_views` = 5, and `low_consensus_threshold` = 3)
         - And this method will also create Caesar Subject Rules and Effects needed for ALICE with defaults.
         (In particular, Question-based retirement's retirement limit is 3 and Count-based retirement default is 30.)
 
         """
+        self.add_to_caesar(public_extracts=True, public_reductions=True)
```

Comment:
FYI: if workflow already exists, the `public_extracts` and `public_reductions` will remain as-is (likely `False` by default).

You could add an if statement to check whether it exists, and if no, create with public extracts and reductions set to `True`, else check setting to make sure extracts and reductions are `True`.

Reply:
Oh fair point. I haven't created any update calls. To clarify, (since I havent really introduced caesar update methods to python client), are you saying, with this check, if workflow exists, send warning to check setting of public extracts and reductions? or are you saying that if a workflow exists, update public_extracts, public_reductions to true?

Reply:
The latter: if a workflow exists, update public_extracts + public_reductions to true (if they aren't already).

Reply:
ah sounds good. if its ok, do you mind if i merge this in and add this ^ on as a subsequent PR?

Reply:
Yes, that works for me!